docs/contivctl.md

```diff
@@ -0,0 +1,226 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+Command can be found under $GOPATH/bin directory (/opt/gopath/bin in vagrant VMs) after a build. 
+Currently, contivctl needs to be run on the same host where netmaster is running
+
+## Usage
+
+```
+vagrant@netplugin-node1:~$ contivctl --help
+usage: contivctl [-h] [--version] {policy,rule,group,network} ...
+
+positional arguments:
+  {policy,rule,group,network}
+    policy              Policy operations
+    rule                Rule add/delete
+    group               Endpoint group operations
+    network             Network operations
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --version             show program's version number and exit
+```
+
+### Creating networks
```

Comment:
Thanks. I'll change the heading.

Yeah, i think i need to create a read me for contiv object model so that we can point it to people and refer it from this document